sentry_sdk/integrations/django/caching.py

```diff
@@ -0,0 +1,105 @@
+import functools
+from typing import TYPE_CHECKING
+
+from django import VERSION as DJANGO_VERSION
+from django.core.cache import CacheHandler
+
+from sentry_sdk import Hub
+from sentry_sdk.consts import OP, SPANDATA
+from sentry_sdk._compat import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Callable
+
+
+METHODS_TO_INSTRUMENT = [
+    "get",
+    "get_many",
+]
+
+
+def _patch_cache_method(cache, method_name):
+    # type: (CacheHandler, str) -> None
+    from sentry_sdk.integrations.django import DjangoIntegration
+
+    def _instrument_call(cache, method_name, original_method, args, kwargs):
+        # type: (CacheHandler, str, Callable[..., Any], Any, Any) -> Any
+        hub = Hub.current
+        integration = hub.get_integration(DjangoIntegration)
+        if integration is None or not integration.cache_spans:
+            return original_method(*args, **kwargs)
+
+        description = "{} {}".format(method_name, " ".join(args))
+
+        with hub.start_span(op=OP.CACHE, description=description) as span:
+            value = original_method(*args, **kwargs)
+
+            if value:
```

Comment:
@antonpirker you can't call `__bool__` of the returned value, it might not be implemented. You should check if a default value was returned, in which case it is a cache miss, or if there is not default value configure you should check `value is not None`.  

I recently reported the same issue on `ddtrace`, you can check of examples on how to reproduce there https://github.com/DataDog/dd-trace-py/issues/5460 and here is how they handle it https://github.com/DataDog/dd-trace-py/pull/5477